### PR TITLE
added rgba-hex plugin

### PR DESCRIPTION
**Pull request by XOP**
plugin converts rgb/rgba values to hex analogues.
currently functionality is limited, but it's the matter of time.
plans for nearest future - add bg color mix support

### CODE REVIEW

**Comment by TrySound** [comment]
@ai Something familiar)))

**Comment by TrySound** [comment]
@XOP Why you added link twice?

**Comment by XOP** [comment]
@TrySound I've just replaced quotes with correct symbols, there shouldn't be any clones

File: README.md [hunk 1]
```diff
@@ -639,6 +640,7 @@ See also plugins in modular minifier [`cssnano`].
 [`postcss-fallback`]:                https://github.com/MadLittleMods/postcss-fallback
 [`postcss-imperial`]:                https://github.com/cbas/postcss-imperial
 [`postcss-position`]:                https://github.com/seaneking/postcss-position
+[`postcss-rgba-hex`]:                https://github.com/XOP/postcss-rgba-hex
```
Comment (TrySound) [inline README.md]: here

File: README.md [hunk 2]
```diff
@@ -646,6 +648,7 @@ See also plugins in modular minifier [`cssnano`].
 [`pleeease-filters`]:                https://github.com/iamvdo/pleeease-filters
 [`postcss-fontpath`]:                https://github.com/seaneking/postcss-fontpath
 [`postcss-reporter`]:                https://github.com/postcss/postcss-reporter
+['postcss-rgba-hex']:                https://github.com/XOP/postcss-rgba-hex
```
Comment (TrySound) [inline README.md]: and here

**Comment by XOP** [comment]
right, found, fixed

**Comment by TrySound** [comment]
@XOP And now squash please

**Comment by ai** [comment]
@XOP can yoou make 1 commit (squash)? Just `git reset HEAD^^^; git commit …; git push origin master -f`

**Comment by himynameisdave** [comment]
Safe to add to the [`postcss-plugins`](https://github.com/himynameisdave/postcss-plugins) list?

Also PS: [One of my plugins](https://github.com/himynameisdave/postcss-rgb-plz) does the exact opposite of this lol :smile:

**Comment by TrySound** [comment]
Partially it can be done with https://github.com/jonathantneal/postcss-unrgba

**Comment by ai** [comment]
@XOP do you need any help with squash?

**Comment by XOP** [comment]
@ai, @TrySound sorry guys, have been out of internet coverage, thanks for help, will be back soon :)

**Comment by XOP** [comment]
:up:

**Pull request merged**